### PR TITLE
Feature/disable task title formatting

### PR DESCRIPTION
**Pull request by naima-shk**
closes [#1881](https://github.com/taskcluster/taskcluster/issues/1881)
@helfi92

### CODE REVIEW

File: ui/src/views/Tasks/ViewTask/index.jsx [hunk 1]
```diff
@@ -803,6 +803,7 @@ export default class ViewTask extends Component {
       <Dashboard
         title={task ? task.metadata.name : 'Task'}
```
Comment (helfi92) [inline ui/src/views/Tasks/ViewTask/index.jsx]:
```suggestion
         title={task ? `Task "${task.metadata.name}"` : 'Task'}
```

File: changelog/issue-1881.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+level: silent
```
Comment (helfi92) [inline changelog/issue-1881.md]:
```suggestion
level: patch
```

File: changelog/issue-1881.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+reference: issue 1881
+---
```
Comment (helfi92) [inline changelog/issue-1881.md]:
```suggestion
---
Taskcluster UI now properly renders the task title in the app bar.
```

**Comment by naima-shaikh** [comment]
@helfi92  Please update review :)

**Pull request merged**